contracts/optimistic-ethereum/OVM/execution/OVM_StateManager.sol

```diff
@@ -578,11 +606,69 @@ contract OVM_StateManager is iOVM_StateManager {
         return totalUncommittedContractStorage;
     }
 
+    function wasContractStorageChanged(
```

Comment:
Ditto for account comment above that this does not return `true` for `ITEM_COMMITTED`